tonic_validate/validate_scorer.py

```diff
@@ -40,11 +44,17 @@ def __init__(
             The list of metrics to be used for scoring.
         model_evaluator: str
             The model to be used for scoring.
+        max_parsing_retries: int
+            The number of times to retry a failed score if it failed due to parsing.
+        max_llm_retries: int
+            The number of times to retry a failed llm request.
         fail_on_error: bool
             If True, an error in calculating a metric will raise an exception. If False, the score will be set to None.
         """
         self.metrics = metrics
         self.model_evaluator = model_evaluator
+        self.max_scoring_retries = max_parsing_retries
```

Comment:
Why isn't the naming consistent?

Reply:
This variable, `self.max_scoring_retries` is never used, so it along with the parameter `max_parsing_retries` should be removed.

Reply:
See the comment below. I think you forgot to use this variable.